numpy/core/_add_newdocs.py

```diff
@@ -4501,38 +4502,30 @@
     """
     a.transpose(*axes)
 
-    Returns a view of the array with axes transposed.
+    Return a view of the array with axes reversed or permuted.
```

Comment:
I think I prefer Returns, and also the permuted keyword is a bit confusing.

Reply:
Regarding Return vs. Returns, the docs seem to be inconsistent

```bash
$ grep -Pn '^ +Return ' numpy/core/_add_newdocs.py | wc -l
42
$ grep -Pn '^ +Returns ' numpy/core/_add_newdocs.py | wc -l
19
```

I was motivated by PEP257, which suggests

> The docstring is a phrase ending in a period. It prescribes the function or method’s effect as a command (“Do this”, “Return that”), not as a description; e.g. don’t write “Returns the pathname …”.

But, I'm happy to revert this change.

Regarding "permuted", I used it to unify it with `np.transpose`, but I also actually prefer "transposed".

Reply:
I think I'd like 'transposed' in both cases, and I'm a bit on the fence about `returns` vs `return` in this scenario.

Lets go with `Return` and `transposed`.

Reply:
Ok, I pushed the change.